install/docker-extension/ui/src/components/ExtensionComponent/ExtensionComponent.js

```diff
@@ -132,6 +133,36 @@ const ExtensionsComponent = () => {
       .catch(console.error)
   }
 
+  const onSubmit = async feedback => {
+    // if (!validateFeedback(feedback)) {
+    //   handleError(
+    //     "We are unable to process your feedback. Did you include a message in your submission?"
+    //   );
+    //   return;
+    // }
+    // const path = router.pathname;
+    const userFeedbackRequestBody = {
+      scope: feedback?.label,
+      message: feedback?.message,
+      page_location: "",
+      metadata: {}
+    };
+    fetch(proxyUrl + '/api/extensions/identity/users/notify/feedback', { 
```

Comment:
The api giving error of cannot `Error: Status Code: 405.failed to request to remote provider`.
Am i missing something? // @humblenginr @MUzairS15

Reply:
do not use `fetch` directly. Use `window.ddClient`.